pkg/sqlcache/informer/listoption_indexer.go

```diff
@@ -790,7 +790,8 @@ func (l *ListOptionIndexer) constructQuery(lo *sqltypes.ListOptions, partitions
 	} else {
 		// make sure one default order is always picked
 		if l.namespaced {
-			query += "\n  ORDER BY f.\"metadata.namespace\" ASC, f.\"metadata.name\" ASC "
+			// ID == metadata.namespace + "/" + metaqata.name
+			query += "\n  ORDER BY f.\"id\" ASC "
```

Comment:
I know the original issue says `id` column but I honestly thought it would be the `key` column which is the name+namespace already. Isn't that what Silvio meant originally?

Otherwise, I don't see where this id column comes from. Objects received from k8s don't have such a field.